app/jsx/components/JumpComp.jsx

```diff
@@ -15,16 +15,18 @@ class JumpComp extends React.Component {
       <div className="c-jump">
         <Link id="c-jump__label" to="#" onClick={(e)=>this.handleClick(e, 1)}>Article</Link>
         <ul className="c-jump__tree" aria-labelledby="c-jump__label">
+       { this.props.attrs.abstract &&
           <li><Link to="#">Abstract</Link></li>
+       }
```

Comment:
It took me a while, but I get what you're doing with the indentation here. Trying to preserve mergeability with Joel's code right? Clever.